jme3-examples/build.gradle

```diff
@@ -20,12 +20,9 @@ dependencies {
     implementation project(':jme3-jbullet')
     implementation project(':jme3-jogg')
     implementation project(':jme3-lwjgl')
-//    implementation project(':jme3-lwjgl3')
     implementation project(':jme3-networking')
     implementation project(':jme3-niftygui')
     implementation project(':jme3-plugins')
-//    implementation project(':jme3-plugins-json')
```

Comment:
I commented these out as part of PR #2278. I'm hoping they can be put back in after somebody figures out why the jme3-plugins JSON implementation wasn't working.